Packet++/header/RawPacket.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <stdint.h>
+#include <memory>
```

Comment:
Why is this include needed? 🤔

Reply:
Its not. It's left over from when I was prototyping the clone method as i was initially thinking of using unique_ptr but that had issues with unique pointers of Derived types not being covariant of unique pointers to base types.